packages/strapi-plugin-content-manager/admin/src/components/CustomTable/index.js

```diff
@@ -3,28 +3,26 @@ import PropTypes from 'prop-types';
 import { withRouter } from 'react-router';
```

Comment:
I think that we should remove this HoC and use the hooks instead to retrieve the `pathname` and `search`.